services/cookie.js

```diff
@@ -0,0 +1,30 @@
+import { config } from '~/config/environment'
+
+function getCookie(name) {
+  const cookie = document.cookie
+    .split('; ')
+    .find((item) => item.startsWith(name))
+
+  if (!cookie) return null
+
+  return cookie.split('=')[1]
+}
+
+function setCookie(name, value, isDev) {
+  if (!name || !value) return
+
+  const localeCookieProperties = [
+    `${name}=${value}`,
+    'path=/',
+    'max-age=31536000',
+    'SameSite=Strict',
+  ]
+
+  if (!isDev) {
+    localeCookieProperties.push(`domain=${config.siteDomain}`)
+  }
+
+  document.cookie = localeCookieProperties.join('; ')
+}
```

Comment:
Super ce cookie service ❤️

Juste quelques suggestions, non-bloquantes donc :

1. Je trouve que protéger l'utilisation de la fonction contre un `name` _falsy_ ou une `value` _falsy_ laisse penser que ce sont des usages qui existent dans le code du projet. Or ces usages n'existent pas et n'arriveront clairement pas. Au besoin un jour on pourra tout au plus faire un `deleteCookie`.
2. Ce service étant générique, nommons les propriétés de manière générique

```suggestion
function setCookie(name, value, isDev) {
  const cookieProperties = [
    `${name}=${value}`,
    'path=/',
    'max-age=31536000',
    'SameSite=Strict',
  ]

  if (!isDev) {
    cookieProperties.push(`domain=${config.siteDomain}`)
  }

  document.cookie = cookieProperties.join('; ')
}
```

Reply:
En effet, j'avais mis ce check en pensant que cette fonction allait être un utilitaire. Ce qui n'est pas le cas vu qu'on importe `config`. 
C'est retiré. J'en ai aussi profité pour mettre à jour le nom de la variable `cookieProperties` 😄